### PR TITLE
UI :- Change the Announcement card style as per new mocs

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes :

Change Announcement card ui as per new mocs

#
### Type of change :

- [x] Improvement

#
### Frontend Preview (Screenshots) :
image

image

### CODE REVIEW

**Comment by harshach** [comment]
@devyani-kaushik that "Announcement" text is not readable, might be better to use different color or do we want to put that orange/yellow color background or just the line like rest of the tasks

**Review by ShaileshParmar11 — APPROVED** [review]

File: openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/Shared/Badge.less [hunk 1]
```diff
@@ -1,17 +1,34 @@
...
 .announcement-badge {
-  width: 24px;
-  height: 24px;
-  background: #ffc143;
-  border: 1px solid #ffc143;
-  border-radius: 50%;
-  padding: 4px;
+  width: 12px;
+  height: 12px;
+  background: #fdfbfd;
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/Shared/Badge.less:17]: Can we use `less CSS` variables for colours?

File: openmetadata-ui/src/main/resources/ui/src/styles/x-master.css [hunk 2]
```diff
@@ -1323,8 +1323,13 @@ div.ant-typography-ellipsis-custom {
 }
 
 /* Feed text pre tag CSS */
+.activity-feed-card-text {
+  padding-top: 6px;
+}
+
 .activity-feed-card-text pre {
   white-space: pre-wrap; /* Since CSS 2.1 */
+  padding-top: 10px;
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/styles/x-master.css:1332]: Why we are adding padding-top to `pre` tag?

File: openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/Shared/Badge.less (hunk 1 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/Shared/Badge.less]: Updating the change.

File: openmetadata-ui/src/main/resources/ui/src/styles/x-master.css (hunk 2 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/styles/x-master.css]: added for testing styles, will remove it.

**Pull request merged**